README.md

```diff
@@ -36,15 +36,18 @@ _✨ A modern Anki [custom scheduling](https://faqs.ankiweb.net/the-2021-schedul
 
 # Introduction
 
-FSRS4Anki consists of two main parts: the scheduler and the optimizer.
-
-- The scheduler replaces Anki's built-in scheduler and schedules the cards according to the FSRS algorithm.
-- The optimizer uses machine learning to learn your memory patterns and finds parameters that best fit your review history. For details about the working of the optimizer, please read [the mechanism of optimization](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-mechanism-of-optimization).
+FSRS has been integrated into Anki. If your version of Anki is 23.10 or newer, use the built-in FSRS. Read about how to use it [here](https://github.com/open-spaced-repetition/fsrs4anki/README2.md).
+If your version of Anki is 2.1.66 or older, use the standalone version.
```

Comment:
The introduction section serves as an introduction to FSRS in general. It is not specific to the standalone version. So, I think that this message should be added to the "How to Get Started?" section.

Reply:
Also, I think that my version of the update notification is more concise: https://github.com/open-spaced-repetition/fsrs4anki/pull/494#issuecomment-1763276719